### ISSUE TITLE
Configurable URL

### ISSUE THREAD

**Issue by ianb** [jnordberg/irccloudapp#28]
We have a company instance of IRCCloud hosted at irccloud.mozilla.com – if I could configure the URL then I could use it with Nimbus (I believe it's otherwise identical to irccloud.com)

**Comment by jnordberg** [comment]
I remember adding this to test irrcloud beta versions which where served on beta.irccloud.com. Try `defaults write com.yellowagents.nimbus url -string "https://irccloud.mozilla.com"`

**Comment by robhudson** [comment]
Should this still work? I've set the `url` as shown and see it when I run `defaults read com.yellowagents.nimbus` but it doesn't seem to load the URL.

**Comment by jnordberg** [comment]
It should: https://github.com/jnordberg/irccloudapp/blob/master/AppDelegate.m#L37

Make sure the app is not running when you write the defaults, sometimes they get overwritten.

**Comment by robhudson** [comment]
Yep, did that. I verified that even after it is launched the url is still set. But it loads the real irccloud, not the URL I'm providing.

**Comment by jnordberg** [comment]
Found the problem, the app id should be capitalized, like so:
`defaults write com.yellowagents.Nimbus url -string "https://irccloud.mozilla.com"`

**Comment by robhudson** [comment]
:bomb: That worked. Thanks!
